client/state/action-types.js

```diff
@@ -109,7 +109,9 @@ export const SERVER_DESERIALIZE = 'SERVER_DESERIALIZE';
 export const SET_EXPORT_POST_TYPE = 'SET_EXPORT_POST_TYPE';
 export const SET_ROUTE = 'SET_ROUTE';
 export const SET_SECTION = 'SET_SECTION';
+// TODO(mcsf): CHANGE WORD ORDER OF THESE TWO:
```

Comment:
Adopt the same schema as other action types, where the object precedes the verb: `GUIDESTOUR_SHOW`, `~_UPDATE`. These can be changed when we change the name to `guided-tours`.